sql_generators/serp_events/templates/metadata.yaml

```
@@ -11,6 +11,12 @@ labels:
 scheduling:
   dag_name: bqetl_serp
   task_name: {{ app_name }}_serp_events__v1
+  date_partition_parameter: null
+  destination_table:
+    >-
+    {{ app_name }}_serp_events_v1${{
```

```suggestion
    serp_events_v1${{
```
are the table names prefixed by the app name? Looks like it's just `serp_events_v1` here: https://github.com/mozilla/bigquery-etl/blob/f41c85a37d184aa2e959a2d5fb580cce729005dc/sql_generators/serp_events/__init__.py#L59